.verdaccio/config.yml

```diff
@@ -0,0 +1,28 @@
+# path to a directory with all packages
```

Comment:
remove, will be handled as part of https://linear.app/openops/issue/OPS-1931/update-create-block-cli-command